style/elixir/README.md

```diff
@@ -0,0 +1,11 @@
+Elixir
+======
+
+[Sample](sample.ex)
+
+* Avoid macros.
+* Don't separate multiple declarations of the same function with a newline.
```

Comment:
Is this a common thing in the community? I haven't followed this pattern so I'm hesitant on giving it a thumb without seeing it in action.